data/json/items/vehicle/plating.json

```diff
@@ -177,5 +177,22 @@
     "description": "Durable silica-coated chitin plating made for a vehicle.",
     "material": [ "acidchitin" ],
     "proportional": { "price": 1.333, "weight": 1.2, "volume": 1.18, "bashing": 1.25 }
+  },
+  {
+    "type": "GENERIC",
+    "id": "counterweight",
+    "name": { "str": "counterweight" },
+    "description": "A heavy block of steel used in balancing other heavy things.",
+    "weight": "750000 g",
+    "to_hit": -10,
+    "color": "dark_gray",
+    "symbol": "]",
+    "material": [ "steel" ],
+    "volume": "20 L",
```

Comment:
Did you also get the dimensions for the counterweight, as this is much denser than our steel ingots 🤔

Reply:
I have to admit that I didn't because I didn't think that would matter, so I just copied the volume of a heavy duty frame.
However since it does, I went ahead and took measurements of the counterweight in my forklift and came with a number of 660 liters, which I divided by 2 because there are no multi-tile parts as far as I know.

I also searched around on what material they are made out of which came up with either cast-iron or concrete, iron being more commonly used.

I also did more research on how to install and uninstall those counterweights and it turns out you only need a wrench rather than a whole welding setup.